### PR TITLE
Add circle annotation cesium

### PR DESCRIPTION
**Pull request by darknath10**

### CODE REVIEW

File: imagery-cesium/plugins/draw-annotations.visualizer.ts [hunk 1]
```diff
@@ -263,6 +283,12 @@ export class CesiumDrawAnnotationsVisualizer extends BaseImageryPlugin {
 				}
 				break;
 			}
+			case AnnotationMode.Circle: {
+				const center = this.startDrawingScreenPixels;
+				const radius = Math.sqrt(Math.pow(center.x - this.lastDrawingScreenPixels.x, 2) + Math.pow(center.y - this.lastDrawingScreenPixels.y, 2));
```
Comment (Suzizaher) [inline imagery-cesium/plugins/draw-annotations.visualizer.ts:288]: Maybe we can add distance function in utils

File: imagery-cesium/plugins/draw-annotations.visualizer.ts [hunk 2]
```diff
@@ -280,4 +306,44 @@ export class CesiumDrawAnnotationsVisualizer extends BaseImageryPlugin {
...
+	}
+
+	/**
+	 * 
+	 * @param radius in pixels
+	 * @param center in pixels
+	 * @param rotation in degrees - optional - default 0
+	 */
+	private createCircleGeometry(radius: number, center: Cartesian2, rotation = 0): Geometry {
+		rotation = rotation / 180 * Math.PI;
```
Comment (Suzizaher) [inline imagery-cesium/plugins/draw-annotations.visualizer.ts:323]: I think we can remove the rotation if we're not using it

File: imagery-cesium/plugins/draw-annotations.visualizer.ts [hunk 3]
```diff
@@ -280,4 +306,44 @@ export class CesiumDrawAnnotationsVisualizer extends BaseImageryPlugin {
...
+		let n = 36 // n sampling angles
+		let coords = [];
+		const cartesia3 = [];
+
+		for (let i = 0; i <= n; i++) {
+		  // get the current angle
+		  const θ = Math.PI * 2 / n * i + rotation;
+
+		  // get the radius at that angle
+		  const r = Math.pow(radius, 2) / Math.sqrt(Math.pow(radius, 2) * Math.sin(θ) * Math.sin(θ) + Math.pow(radius, 2) * Math.cos(θ) *  Math.cos(θ));
```
Comment (Suzizaher) [inline imagery-cesium/plugins/draw-annotations.visualizer.ts:333]: this calculation is relevant only for ellipse! I think we can remove it and use the radius that the function gets as input

File: imagery-cesium/plugins/draw-annotations.visualizer.ts [hunk 4]
```diff
@@ -280,4 +306,44 @@ export class CesiumDrawAnnotationsVisualizer extends BaseImageryPlugin {
...
+		return Cartesian3.distance(center, end);
+	}
+
+	/**
+	 * 
+	 * @param radius in pixels
+	 * @param center in pixels
+	 * @param rotation in degrees - optional - default 0
+	 */
+	private createCircleGeometry(radius: number, center: Cartesian2, rotation = 0): Geometry {
```
Comment (Suzizaher) [inline imagery-cesium/plugins/draw-annotations.visualizer.ts:322]: Think of how we can remove this function somehow to utils

File: imagery-cesium/plugins/draw-annotations.visualizer.ts [hunk 5]
```diff
@@ -280,4 +306,44 @@ export class CesiumDrawAnnotationsVisualizer extends BaseImageryPlugin {
...
+		const cartesia3 = [];
+
+		for (let i = 0; i <= n; i++) {
+		  // get the current angle
+		  const θ = Math.PI * 2 / n * i + rotation;
+
+		  // get the radius at that angle
+		  const r = Math.pow(radius, 2) / Math.sqrt(Math.pow(radius, 2) * Math.sin(θ) * Math.sin(θ) + Math.pow(radius, 2) * Math.cos(θ) *  Math.cos(θ));
+
+		  // get the x,y coordinate that marks the ellipse at this angle
```
Comment (Suzizaher) [inline imagery-cesium/plugins/draw-annotations.visualizer.ts:335]: remove comment of fix it

File: imagery-cesium/plugins/draw-annotations.visualizer.ts (hunk 2 above)
Comment (asafmas-rnd) [inline imagery-cesium/plugins/draw-annotations.visualizer.ts:323]: use toDegrees and toRadians methods from imagery

File: imagery-cesium/plugins/draw-annotations.visualizer.ts (hunk 3 above)
Comment (asafmas-rnd) [reply imagery-cesium/plugins/draw-annotations.visualizer.ts:333]: try to use http://turfjs.org/docs/#circle
or this http://turfjs.org/docs/#destination
for the calculation

add the methods to imagery\utils\geo

**Pull request merged**